.github/workflows/test.yml

```diff
@@ -23,3 +23,20 @@ jobs:
       uses: jbangdev/jbang-action@v0.111.0
       with:
         script: test.java
+  jbang-drools-snapshot:
+    runs-on: ubuntu-latest
+    name: Testing with Drools latest SNAPSHOT release
+    steps:
+      - name: checkout
+        uses: actions/checkout@v1
+      - uses: actions/cache@v1
+        with:
+          path: /root/.jbang
+          key: $-jbang-$
+          restore-keys: |
+            $-jbang-
+      - name: jbang
+        uses: jbangdev/jbang-action@v0.111.0
+        with:
+          script: test.java
+          jbangargs: -Ddrools.version=999-SNAPSHOT
```

Comment:
for the next iteration, you can consider making this a GHA-matrix parameter and define a parametrized GHA-job